themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+          "enableDnsSupport": true,
+          "id": "vpc-04057e18647a66e24",
+          "instanceTenancy": "default",
+          "ipv6AssociationId": "",
+          "ipv6CidrBlock": "",
+          "mainRouteTableId": "rtb-01c36ce5fcc3ac797",
+          "ownerId": "0123456789",
+          "tags": {}
+        },
+        "parent": "urn:pulumi:dev-user1::demo-aws-ts-webserver::pulumi:pulumi:Stack::demo-aws-ts-webserver-dev-user1",
+        "provider": "urn:pulumi:dev-user1::demo-aws-ts-webserver::pulumi:providers:aws::default_3_32_0::d77c9fd7-1c06-427d-ba26-dbd8a8cbf91a",
+        "propertyDependencies": {
+          "cidrBlock": null
+        }
+      }
+    ]
+  }
+}
+```
+<!-- ###################################################################### -->
+
+## Stack Tags
+
+### Get Stack Tags
+
+Use [Get Stack](#get-stack) to retrieve a stack's details including its tags.
+
+### Set Stack Tag
+
+```
+POST https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `team` | string | path | team name |
+| `stack` | string | path | stack name |
+| `name` | string | body | tag name |
+| `value` | string | body | tag value |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+3' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
+  --request POST \
+  --data '{"name":"{tagName}","value":"{tagValue}"}' \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags
+```
+
+#### Default response
+
+```
+Status: 204 OK
+```
+
+```
+EMPTY RESPONSE BODY
+```
+
+<!-- ###################################################################### -->
+
+### Delete Stack Tag
+
+```
+DELETE https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags/{tagName}
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `team` | string | path | team name |
+| `stack` | string | path | stack name |
+| `tagName` | string | path | tag name |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+3' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
+  --request DELETE \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags/{tagName}
+```
+
+#### Default response
+
+```
+Status: 204 OK
+```
+
+```
+EMPTY RESPONSE BODY
+```
+
+## Stack Updates
+
+### List Stack Updates
+
+```
+GET https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/updates
+```
+
+By default the results are not paginated. You can specify `page` and `pageSize` query parameters to paginate the results.
+> `?pageSize=1&page=1` can be used to return only the most recent update.
```

Comment:
If you want to get fancy, you can just use `"/{stackName}/updates/latest` instead. As that will do the same thing (but return just the latest update, not an array of updates with one element).